<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0" connectslotsbyname="false">
 <class>ModSelector</class>
 <widget class="QWidget" name="ModSelector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ModSelector</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>3</number>
         </property>
         <item>
          <widget class="QLabel" name="label_disabled">
           <property name="text">
            <string>Available items</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_disabledCount">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>(420)</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_filterDisabled">
         <property name="maxLength">
          <number>255</number>
         </property>
         <property name="placeholderText">
          <string>Search...</string>
         </property>
         <property name="clearButtonEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListView" name="list_disabledMods">
         <property name="dragDropMode">
          <enum>QAbstractItemView::DragOnly</enum>
         </property>
         <property name="defaultDropAction">
          <enum>Qt::IgnoreAction</enum>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="iconSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
         <property name="verticalScrollMode">
          <enum>QAbstractItemView::ScrollPerPixel</enum>
         </property>
         <property name="movement">
          <enum>QListView::Free</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button_enable">
         <property name="toolTip">
          <string>Enable the selected mods for the current configuration</string>
         </property>
         <property name="text">
          <string>Enable selected →</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="spacing">
          <number>3</number>
         </property>
         <item>
          <widget class="QLabel" name="label_enabled">
           <property name="text">
            <string>Enabled items</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_enabledCount">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>(69)</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_filterEnabled">
         <property name="maxLength">
          <number>255</number>
         </property>
         <property name="placeholderText">
          <string>Search...</string>
         </property>
         <property name="clearButtonEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QListView" name="list_enabledMods">
           <property name="dragDropMode">
            <enum>QAbstractItemView::DragOnly</enum>
           </property>
           <property name="defaultDropAction">
            <enum>Qt::IgnoreAction</enum>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::ExtendedSelection</enum>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
           <property name="iconSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
           <property name="verticalScrollMode">
            <enum>QAbstractItemView::ScrollPerPixel</enum>
           </property>
           <property name="movement">
            <enum>QListView::Free</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <property name="sizeConstraint">
            <enum>QLayout::SetFixedSize</enum>
           </property>
           <item>
            <widget class="QPushButton" name="button_moveUp">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>25</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="toolTip">
              <string>Move selected mods up to increase priority during game start</string>
             </property>
             <property name="text">
              <string>↑</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="button_moveDown">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>25</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="toolTip">
              <string>Move selected mods down to decrease priority during game start</string>
             </property>
             <property name="text">
              <string>↓</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="button_disable">
         <property name="toolTip">
          <string>Disable the selected mods for the current configuration</string>
         </property>
         <property name="text">
          <string>← Disable selected</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="widget_modDetails" native="true"/>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEdit_filterDisabled</tabstop>
  <tabstop>list_disabledMods</tabstop>
  <tabstop>button_enable</tabstop>
  <tabstop>lineEdit_filterEnabled</tabstop>
  <tabstop>list_enabledMods</tabstop>
  <tabstop>button_disable</tabstop>
  <tabstop>button_moveUp</tabstop>
  <tabstop>button_moveDown</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
